src/audio/playback.cpp

```diff
@@ -0,0 +1,26 @@
+#include "playback.h"
+
+Playback::Playback() {
+    is_playing = false;
+    volume = 1.0f;      // Default volume 100%
+}
+
+Playback::~Playback() {
+    // Destructor
+}
+
+void Playback::play() {
+    // We will have to implement further logic from here on
+    is_playing = true;
+    std::cout << "Playback started." << std::endl;
+}
+
+void Playback::pause() {

```

Comment:
Is it worth having pause and play? I feel like they effectively do the same thing. Maybe we can just have Playback::stop

Reply:
I named them this way because stop gives me the connotation that it won't maintain state and it won't be able to start playing again from the stop point. It can easily be called stopped as long as it won't give off the wrong connotations